### ISSUE TITLE
Does JB Fiddles support multiple layers?

### ISSUE THREAD

**Issue by csobier** [CartoDB/labs-cdbfiddle#2]
I cannot view any of my CartoCSS code for this map?  (It contains four layers).

@chriswhong , does JB Fiddles support multiple layers? If not, can you add support to do this?  Please :-)  If not, is there a workaround I can use?

http://cartodb.github.io/labs-cdbfiddle/#https://documentation.cartodb.com/api/v2/viz/ac15e516-8cb0-11e5-9caf-0ea31932ec1d/viz.json

**Comment by chriswhong** [comment]
No, it was meant to show only the first layer for simplicity's sake.  Would
run out of screen real estate quickly. I'll grab you in the office to see
what you have in mind.

**Comment by csobier** [comment]
Okay, bummer, but makes sense.  Let me know if there is another way to show a map and CartoCSS code?

**Comment by csobier** [comment]
As requested, see sample screenshot of the map I would like to show and the associated CartoCSS code (from layer 3).  Ideally, the code would not be an image so users can copy and paste it (and also view the code from each layer. However, I would be happy if I could at least display code from one selected layer).  

I suppose I could create another text element displaying the CartoCSS code on the map itself, but that would be a last resort.

![image](https://cloud.githubusercontent.com/assets/13319925/11216558/c83070d6-8d19-11e5-8468-ff1c17c60a8e.png)
